Dockerfile

```diff
@@ -0,0 +1,39 @@
+
+from node:lts as builder
+workdir hubs
+copy . .
+env BASE_ASSETS_PATH="{{rawhubs-base-assets-path}}"
+run npm ci 
+run npm run build
+run cd admin && npm ci && npm run build && cp -R dist/* ../dist && cd ..
+run mkdir -p dist/pages && mv dist/*.html dist/pages && mv dist/hub.service.js dist/pages && mv dist/schema.toml dist/pages          
+#[info] rearrange files
+run mkdir rawhubs && mv dist/pages rawhubs && mv dist/assets rawhubs && mv dist/react-components rawhubs/pages && mv dist/favicon.ico rawhubs/pages
+
+from alpine/openssl as ssl
+run mkdir /ssl && openssl req -x509 -newkey rsa:2048 -sha256 -days 36500 -nodes -keyout /ssl/key -out /ssl/cert -subj '/CN=hubs'
```

Comment:
What is this for? I guess this is for ssl between reticulum and the hubs source assets, though these are self signed certs so I guess it has to ignore their validity? Is this just to ensure encryption over the wire? These are public assets anyway I think, so this nginx service can probably just be running http. I might also just be missing something obvious.

Reply:
long story short / if i remembered correctly -- it's for ret, it only wants assets served on https
we technically can terminate https on ingress (fraefik / haproxy) otherwise, i just didn't want to bother with more ret code changes lol

Reply:
I think changing ret to remove the ssl requirement is worth it later on, but it's okay to do this for now.